psm_utils/peptidoform.py

```diff
@@ -296,8 +320,9 @@ def sequential_theoretical_mass(self) -> float:
         """
         fixed_rules = {}
         for rule in self.properties["fixed_modifications"]:
-            for aa in rule.targets:
-                fixed_rules[aa] = rule.modification_tag.mass
+            if rule.targets is not None:
```

Comment:
There are a couple of these I see, but personally I would probably have done `if rule.targets is None: continue` instead, avoiding the extra indent level.